html-forms/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      HTML forms
+    </title>
+  </head>
+  <body>
+    <h1><strong>Coding Form!</strong></h1>
+    <form>
+      <div><label for="email">Email:</label></div>
+      <div><input type="text" name="email" size="30"></div>
```

Comment:
The value of the `id` attribute on the `input` needs to match the value of the `for` attribute on the `label` it corresponds with.